Dockerfile

```diff
@@ -5,4 +5,4 @@ COPY package.json /usr/src/app/
 RUN npm install
 COPY . /usr/src/app
 EXPOSE 3000
-CMD [ "./utils/wait-for-it.sh", "mongo:27017", "--", "npm", "start" ]
```

Comment:
`wait-for-it.sh` was needed to wait for Mongo to be ready before starting the main server.
https://medium.com/@selahattinunlu/how-to-wait-for-a-container-to-be-ready-757bc1a86468

Is there a specific reason this is not needed anymore?

Reply:
@harlev I have added a logic in node.js server to retry if the database is not connected. So this is not needed anymore.